components/pipeline-service/base/rbac/cluster-role/pipeline-service-sre.yaml

```diff
@@ -4,14 +4,22 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: pipeline-service-sre
 rules:
-  # This rule is needed until RHTAPBUGS-256 is fixed
+  # This rule is needed to deal with situations like RHTAPBUGS-256, where we followed up via PLNSRVCE-1328
```

Comment:
Keep the comment for secrets list/deletion.
Put your comment before the new rbac you are adding,
Add secrets to the new rbac.

When OSP 1.12 is deployed, we'll removed the old RBAC, which will remove secret deletion.

Reply:
updates pushed @Roming22 thanks